src/es/components/molecules/carouselTwo/CarouselTwo.js

```diff
@@ -573,6 +584,28 @@ export default class CarouselTwo extends Mutation() {
           path: `${this.importMetaUrl}./seperate-nav-/seperate-nav-.css`, // apply namespace since it is specific and no fallback
           namespace: false
         }, ...styles], false).then(() => setAttributeStyles())
+
+      case 'carousel-two-3-column-':
+        return this.fetchCSS([{
+          path: `${this.importMetaUrl}./default-/default-.css`, // apply namespace since it is specific and no fallback
+          namespace: false,
+          replaces: [{
+            pattern: '--carousel-two-default-',
+            flags: 'g',
+            replacement: '--carousel-two-3-column-'
+          }]
+        }, {
+          path: `${this.importMetaUrl}./3-column-/3-column-.css`, // apply namespace since it is specific and no fallback
+          namespace: false
+        }, ...styles], false).then(() => setAttributeStyles())
+      case 'carousel-two-image-':
+        return this.fetchCSS([{
+          path: `${this.importMetaUrl}./default-/default-.css`, // apply namespace since it is specific and no fallback
```

Comment:
don't you need to replace the default namespace, if you extend the default template?

Reply:
Done